packages/bigquery/src/index.js

```diff
@@ -617,7 +618,8 @@ BigQuery.prototype.getJobs = function(options, callback) {
 
   this.request({
     uri: '/jobs',
-    qs: options
+    qs: options,
+    useQueryString: true
```

Comment:
I found the problem of https://github.com/GoogleCloudPlatform/google-cloud-node/issues/2182#issuecomment-294312078 `@google-cloud/bigquery@0.9.2` is broken because of here; wonder if @lukesneeringer actually read my #2182 ? or read the [`request`](https://www.npmjs.com/package/request) doc?

```diff
-    useQueryString: true
+    useQuerystring: true
```

![image](https://cloud.githubusercontent.com/assets/14798161/25068871/4e8f1834-2225-11e7-96d9-911fb4208c02.png)